rcl/include/rcl/arguments.h

```diff
@@ -294,8 +297,8 @@ rcl_arguments_get_param_files(
  * Lock-Free          | Yes
  *
  * \param[in] arguments An arguments structure that has been parsed.
- * \param[out] parameter_overrides Zero or more parameter overrides.
- *   This structure must be finalized by the caller.
```

Comment:
Looks like this is still a valid sentence to leave in the documentation.

Reply:
Yes, good catch! Re-added it in https://github.com/ros2/rcl/pull/508/commits/c1f4c3ce1dc69cf0d29cf1c72873d582efa7a557